### DISCUSSION TITLE
📺 CMU 15-122 命令式计算原理 ⭐️ | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#64] [Announcements]
# 📺 CMU 15-122 命令式计算原理 ⭐️ | HackWay技术学习路线

课程名称： Principles of Imperative Computation

https://hackway.org/docs/cs/freshman/cpp/cs15122

**Comment by complexbb** [comment]
这门课放错了把，这门课是数据结构算法，这门课没有开源，作业要登录，
